web_search_nospace/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
```

Comment:
copy is not needed in `__init__`, no mandatory to remove tho... but pls, replace :copyright: w/ "Coypright"